test/test_helpers.js

```diff
@@ -143,6 +143,7 @@ function getMockEnvironmentAsync(options) {
                         airtable: new Airtable({
                             apiKey: 'key123',
                             endpointUrl: 'http://localhost:' + testServerPort,
+                            requestTimeout: 1000,
```

Comment:
nit: maybe add a comment mentioning this is required for the timeout tests

Reply:
Done.